components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -823,10 +925,19 @@ void RRTMGPRadiation::run_impl (const double dt) {
       real2d tmp2d = subview_2d(m_buffer.tmp2d);
       for (int igas = 0; igas < m_ngas; igas++) {
         auto name = m_gas_names[igas];
-        auto full_name = name + "_volume_mix_ratio";
-
-        // 'o3' is marked as 'Required' rather than 'Computed', so we need to get the proper field
-        auto f = name=="o3" ? get_field_in(full_name) : get_field_out(full_name);
+        auto full_name = (name=="o3" && m_transient_ozone) ? name + "_transient" : name + "_volume_mix_ratio";
```

Comment:
If it hasn't become obvious from previous comments by now, I'm confused by the transient ozone flag. Is this just to allow hooking up prognostic ozone? If so, I vote we remove all of this extra future-proofing logic to simplify this interface.